src/styles/blocks/grid.scss

```diff
@@ -0,0 +1,73 @@
+.grid {
+  display: grid;
+
```

Comment:
think you can add these properties here:
```
.grid {
  grid-template-columns: repeat(12, 1fr);
  column-gap: 28px;
}
```

and remove onSmallDesktop and theDesign from here

Reply:
okay, i've already removed it from theDesign and onLargeScreen ;)